djangocms_blog/models.py

```diff
@@ -164,7 +164,8 @@ def get_absolute_url(self, lang=None):
         )
 
     def __str__(self):
-        return self.safe_translation_getter('name', any_language=True)
+        default = ugettext('BlogCategory (no translation)')
```

Comment:
Why not ugettext_lazy?

Reply:
Cause with the lazy function it's raising:

`TypeError: __str__ returned non-string (type __proxy__)`

Reply:
I see :+1: